ada-parser/src/main/java/com/ucl/ADA/parser/ParserServices.java

```diff
@@ -25,4 +30,13 @@ public ParserServices() {
     public SetMultimap<String, ADAClass> parseRepository(String repositoryPath, Set<String> filePaths) {
         return new ADAParser().getParsedSourceFile(repositoryPath, filePaths);
     }
+
```

Comment:
main function should not be in the service

Reply:
Remove the main function